assignments/arrays.js

```diff
@@ -63,21 +63,34 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+// inventory[id.length -1];
 let lastCar = 0;
-console.log();
+// for (let i = 0; i < inventory.lenght; i++)
+console.log(`${inventory[49].car_make} ${inventory[49].car_model} `);
```

Comment:
totally valid since this works, but what if some cars were added and we didn't know how many for some reason? we could say `let lastCar = inventory[inventory.length-1]` and then console log `lastCar.car_model`. any time you want the last item in an array you'll use that same `array[array.length-1]` format